src/installer/pkg/sfx/Microsoft.NETCore.App/Directory.Build.props

```
@@ -3,7 +3,8 @@
 
   <PropertyGroup>
     <SharedFrameworkName>Microsoft.NETCore.App</SharedFrameworkName>
-    <SharedFrameworkName Condition="'$(PgoOptimize)' != ''">$(SharedFrameworkName).PGO</SharedFrameworkName>
+    <SharedFrameworkName Condition="'$(PgoInstrument)' != ''">$(SharedFrameworkName).PGO</SharedFrameworkName>
+    <IsShipping Condition="'$(PgoInstrument)' != ''">true</IsShipping>
```

We shouldn't mark the package as shipping if it's the PGO-instrumented one.

```suggestion
    <IsShipping Condition="'$(PgoInstrument)' != ''">false</IsShipping>
```